tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -43,11 +43,15 @@ describe('Performing an upgrade', () => {
     const currentVersion = await upgradePage.getCurrentVersion();
     expect(currentVersion).to.include(version.getVersion());
 
+    await browser.refresh(); // load updated code of admin app
```

Comment:
It is interesting to me that we need to refresh the  page here (I assume to get the proper build number) when previously the same value was coming back on line 44 as the old version number without having to refresh the page... :thinking:

Am I missing something here?

Reply:
Yea :) 

When you upgrade, the admin page is still rendered by the old version if we don't refresh (it's an angular app). And the "old version" can't display the new build field that I've just added in this PR (I tried lots of options, but this just seems the easiest one to just remove after this code is merged).

Reply:
This is safe to remove after this code is merged (in a new PR). 
But maybe it's ok if it stays?

Reply:
Ahhh, hah!  I was stuck thinking it was the _value_ of the field that was not getting updated. Did not consider that the field, itself, does not exist until the page is reloaded...

:+1: